keyboards/pinky/pro_micro.h

```diff
@@ -0,0 +1,358 @@
+/*
```

Comment:
I don't think that this file is needed, and should be removed.

Reply:
I removed that file.